maintenance/importConfig.ts

```diff
@@ -282,6 +285,130 @@ function ensureArray(json: any): any[] {
 	return isArray(json) ? json : [json];
 }
 
+function normalizeUnits(unit: string) {
+	if (!unit) return undefined;
+
+	if (/minutes/i.test(unit)) {
+		return "minutes";
+	} else if (/seconds/i.test(unit)) {
+		return "seconds";
+	} else if (/fahrenheit|\bF\b/i.test(unit)) {
```

Comment:
lets make the F lowercase to be consistent ;) I had to think why it was uppercase and the c in the next check was not.